frontend/src/components/Shared/VideoPlayer/VideoPlayer.jsx

```
@@ -2,228 +1,432 @@
+        type="button"
+        tabIndex="0"
+        style={{ display: 'block' }}
+      />
+
+      <div 
+        className={`absolute inset-0 transition-opacity duration-300 bg-gradient-to-t from-black/70 to-transparent flex flex-col justify-end ${showControls || !isPlaying ? 'opacity-100' : 'opacity-0'}`}
+        onClick={(e) => e.stopPropagation()}
+        onKeyDown={(e) => e.stopPropagation()}
+        role="toolbar"
+        aria-label="Video controls"
+        tabIndex="0"
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)